build-linux/make_package.sh

```diff
@@ -131,7 +133,7 @@ else
 fi
 
 echo = FPM!
-_FPM_DEPENDS="--depends sudo --depends python"
```

Comment:
Dropped for the time being since there aren't any consistent providers of just `python` anymore cross distros.

Our currently installation instructions cover installing Python, but if necessary we can add a wrapper or specify more fine-grained dependencies.